htdocs/js/ui/scratchpad.js

```diff
@@ -55,6 +55,14 @@ RCloud.UI.scratchpad = (function() {
                 RCloud.UI.thumb_dialog.init();
 
                 $('#update-thumb').click(function() {
+
+                    // select the thumb in the assets:
+                    var thumb = shell.notebook.model.get_asset('thumb.png');
+
+                    if(thumb) {
+                        thumb.controller.select();
```

Comment:
So you don't see the thumb when you click on the button? _Seems_ to work for me.

Ah - I see! (Or I don't see.) I'll take a look.